packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -545,6 +548,28 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     }).then<int>((_) => count);
   }
 
+  /// Repeatedly triggers `frameCount` frames with a fixed `frameDuration`.
+  /// 
+  /// The `frameDuration` defaults to 16.683 milliseconds (59.94 FPS).
+  /// All frames stop executing after finishing [EnginePhase.build], except for
+  /// the last frame, which executes the full pass.
+  Future<void> pumpFrames(
```

Comment:
Could the docs also provide a sample invocation? I wasn't quite sure how to use it until I looked at #54335